includes/REST_API/Stories_Controller.php

```diff
@@ -320,11 +320,42 @@ public function get_items( $request ) {
 			$posts_query->query( $query_args );
 			$statuses_count[ $key ] = absint( $posts_query->found_posts );
 		}
+
+		if ( $request['_web_stories_envelope'] ) {
+			$current_data = $response->get_data();
+			$headers      = $response->get_headers();
+			$data         = [
+				'totals'  => $statuses_count,
+				'headers' => $headers,
+				'data'    => $current_data,
+			];
+			$response     = rest_ensure_response( $data );
```

Comment:
Can't we just use `\WP_REST_Server::envelope_response()` here? Seems to do basically the same thing.

_At the very least_, change the array structure so that it's the same, to make migration easier once https://github.com/WordPress/gutenberg/issues/23674 is merged.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/2933/commits/1005ffd39f2781976e5737c390d4b212ca099c17